DigitalLearningSolutions.Web/Scripts/searchSortAndPaginate/sort.ts

```diff
@@ -28,8 +28,11 @@ export function getSortValue(
   sortBy: string,
 ): string | number | Date {
   switch (sortBy) {
+    case 'SearchableName':
     case 'Name':
       return getElementText(searchableElement, 'name').toLocaleLowerCase();
+    case 'DateRegistered':
+      return parseDate(getElementText(searchableElement, 'registration-date'));

```

Comment:
It doesn't look like we have any unit tests for this, I can show you where they should go if you need any help with this (.spec.ts files)